addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -120,7 +120,7 @@ public synchronized void playerRewind() {
 
         JsonObject params = new JsonObject();
         params.addProperty("playerid", activePlayer);
-        params.addProperty("speed", "-1");
+        params.addProperty("speed", -2);
```

Comment:
I think it should be -2 because if I use those controls from Kodi UI the JSON-RPC interface sends -2 for REWIND and 2 for FASTFORWARD to OH2 binding.

Kodi accepts different steps for speed. e.g. 2 for 'very slow' FASTFORWARD to 32 'very fast' FASTFORWARD (see [JSON-RPC-API](http://kodi.wiki/view/JSON-RPC_API/v8#Player.SetSpeed), click "show" next to "JSON Schema Description")